src/components/actions/CallToAction.js

```diff
@@ -102,8 +103,6 @@ const CallToAction = ({
       }),
     });
 
-    console.log('resp', resp);
```

Comment:
why the console?

Reply:
That's a line I'm deleting. It was there for local debugging and should have been removed previously.